package.json

```diff
@@ -59,5 +60,8 @@
 	],
 	"engines": {
 		"local-by-flywheel": ">=6.5.2"
+	},
+	"resolutions": {
+		"trim": "0.3.0"
```

Comment:
@joshuapregua I see this when doing `yarn install` now:

```
> yarn
yarn install v1.22.22
[1/5] 🔍  Validating package.json...
warning local-addon-notes@1.2.3: The engine "local-by-flywheel" appears to be invalid.
[2/5] 🔍  Resolving packages...
warning Lockfile has incorrect entry for "trim@0.3.0". Ignoring it.
Couldn't find any versions for "trim" that matches "0.3.0"
? Please choose a version of "trim" from this list: (Use arrow keys)
❯ 1.0.1
  1.0.0
  0.0.3
  0.0.2
```

Did you mean to resolve it to 0.0.3?